digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -173,6 +173,19 @@ object ProjectDAO {
     Q.queryNA[String](query).firstOption
   }
 
+
+  def isNewRoadPartLinkUsed(roadNumber: Long, roadPartNumber: Long, projectId: Long) = {
+    val query =
+      s"""
+		select prol.id
+         from project_link prol
+         join lrm_position pos on prol.lrm_position_id = pos.id
```

Comment:
New version commit will not have this query. As Ossi said is more deeper than that, we need to keep the validation of:
1.new road address should not exist already in Road_address table
2.new road_address (same roadNumber and roadPart) should not exist in Project_link for others project ids